DCP_Tutorial.md

```diff
@@ -0,0 +1,135 @@
+## DCP Demo (self-led)
+**Target Audience:** DCP Staff
+
+**Purpose:** Serve as a learning resource for end-to-end data flow in the DCP
+
+**Components:** Ingest, Upload, Data store, Data pipelines/Analysis, Azul, Data Portal, Matrix
+
+**Reference Document:** https://docs.google.com/document/d/1-9k9tcuIc4aTuxIo-KAT34R_OSca0j9_3LDNJwqksBQ/edit#heading=h.kuwifzq1d62x.
+
+**Note:** This document was published in June 2018 so may have inconsistencies and steps missing. A v2 document will be published soon and linked here.)
+
+### FAQ
+**Who do I contact if I encounter any any issues while following the steps?**
+Feel free reach out to #dcp-ops-help, create an issue in the DCP repository, or ping Parth.
+
+**Do I need to have a certain level of technical proficiency to follow the steps?**
+Technical proficiency should not be a requirement. Please reach out if you encounter any issues, as this indicates that the instructions need additions or modifications.
+
+**How can I make edits to this document?**
+Please create a PR in the DCP repository.
+
```

Comment:
Suggesting making the encouragement even stronger to improve the doc,

Reply:
Removed Please and moved to top of FAQ